src/app/containers/Ad/Amp/index.jsx

```diff
@@ -84,6 +84,7 @@ const AmpAd = ({ slotType }) => {
       amp-access="toggles.ads.enabled"
       amp-access-hide="true"
       aria-hidden="true"
+      aria-label="advertisement"
```

Comment:
Any chance of this being localised? And a different label for the leaderboard to that of the MPU?

Reply:
On AMP we can use aria-labelledby that maps to an id on the advertisement 'label? Though not sure this is possible for canonical...

Reply:
Sorry, could you please clarify for me what exactly should be in the aria label? Do you want the following advertisement label text from configs inside arias?

<img width="355" alt="Screenshot 2020-07-08 at 11 48 46" src="https://user-images.githubusercontent.com/17802955/86910198-2f46df00-c111-11ea-8694-2382a0dff19a.png">
^ that is Pidgin example

Reply:
We should distinguish between the leaderboard and MPU if possible. Is it possible to have for example 'Tori we dem pay for' for the leaderboard and 'Tori we dem pay for **2**' for the MPU as the aria-label?

Reply:
Yes, that sounds good. Will add that now